Manifest.toml

```diff
@@ -598,15 +598,15 @@ version = "1.9.0"
 
 [[deps.HiGHS]]
 deps = ["HiGHS_jll", "MathOptInterface", "PrecompileTools", "SparseArrays"]
-git-tree-sha1 = "2b650f670ad691b94f506e88b4f3ab591a13f809"
+git-tree-sha1 = "f869b0a17d1a4f13aac08af8d0a050bdb70bccfd"
 uuid = "87dc4568-4c63-4d18-b0c0-bb2238e4078b"
-version = "1.8.0"
+version = "1.8.1"
 
 [[deps.HiGHS_jll]]
-deps = ["Artifacts", "CompilerSupportLibraries_jll", "JLLWrappers", "Libdl"]
-git-tree-sha1 = "10bf0ecdf70f643bfc1948a6af0a98be3950a3fc"
+deps = ["Artifacts", "CompilerSupportLibraries_jll", "JLLWrappers", "Libdl", "Zlib_jll"]
+git-tree-sha1 = "f596ee3668df8587158bcaef1ae47bf75bc0fe39"
 uuid = "8fd58aa0-07eb-5a78-9b36-339c94fd15ea"
-version = "1.6.0+0"
+version = "1.6.0+1"
```

Comment:
@SouthEndMusic triggered this manually because this updating HiGHS might fix an issue with the quadratic objective function test that we currently skip. Here we see that the HiGHS library is the same version, 1.6.0, though a newer build, and the wrapper package also updated.

HiGHS 1.6.0 dates back to 2023-09, there have been many commits since, but no release yet. So it could be fixed on the master branch, but we'd have to build it manually.